solitude/exceptions.py

```diff
@@ -2,9 +2,18 @@
 from rest_framework.views import exception_handler
 
 from lib.bango.errors import BangoImmediateError
+from lib.brains.errors import BraintreeResultError
 
 
 def custom_exception_handler(exc):
+    if isinstance(exc, BraintreeResultError):
+        res = {
+            '__all__': [exc.result.message],
+            '__braintree__': 'error',
+            '__type__': 'braintree'
```

Comment:
why do these need to be double underscored? Can you just say ``'error_type': ...`` ?